deploy/runtools/firesim_topology_with_passes.py

```diff
@@ -191,33 +185,22 @@ def pass_no_net_host_mapping(self) -> None:
         servers = self.firesimtopol.get_dfs_order_servers()
         serverind = 0
 
-        for f1_16x in self.run_farm.f1_16s:
-            for x in range(f1_16x.get_num_fpga_slots_max()):
-                f1_16x.add_simulation(servers[serverind])
-                serverind += 1
-                if len(servers) == serverind:
-                    return
-        for f1_4x in self.run_farm.f1_4s:
-            for x in range(f1_4x.get_num_fpga_slots_max()):
-                f1_4x.add_simulation(servers[serverind])
-                serverind += 1
-                if len(servers) == serverind:
-                    return
-        for f1_2x in self.run_farm.f1_2s:
-            for x in range(f1_2x.get_num_fpga_slots_max()):
-                f1_2x.add_simulation(servers[serverind])
+        while len(servers) > serverind:
+            # this call will error if no such instances are available.
+            instance_type = self.run_farm.get_smallest_sim_host_handle(num_sims=1)
+            allocd_instance = self.run_farm.allocate_sim_host(instance_type)
+
+            for x in range(allocd_instance.MAX_SIM_SLOTS_ALLOWED):
+                allocd_instance.add_simulation(servers[serverind])
                 serverind += 1
                 if len(servers) == serverind:
                     return
-        assert serverind == len(servers), "ERR: all servers were not assigned to a host."
 
     def pass_simple_networked_host_node_mapping(self) -> None:
         """ A very simple host mapping strategy.  """
         switches = self.firesimtopol.get_dfs_order_switches()
-        f1_2s_used = 0
-        f1_4s_used = 0
-        f1_16s_used = 0
-        m4_16s_used = 0
+
+        switch_host_inst_type = self.run_farm.get_default_switch_host_handle()
```

Comment:
this should be `switch_host_inst_handle` throughout the code?

Reply:
I wanted to see if "handle" was accepted before changing it everywhere.